book/listings/todo/8/window/mod.rs

```diff
@@ -397,46 +397,37 @@ impl Window {
 
     // ANCHOR: new_collection
     fn new_collection(&self) {
-        // Create new Dialog
-        let dialog = Dialog::with_buttons(
-            Some("New Collection"),
-            Some(self),
-            DialogFlags::MODAL
-                | DialogFlags::DESTROY_WITH_PARENT
-                | DialogFlags::USE_HEADER_BAR,
-            &[
-                ("Cancel", ResponseType::Cancel),
-                ("Create", ResponseType::Accept),
-            ],
-        );
-        dialog.set_default_response(ResponseType::Accept);
-
-        // Make the dialog button insensitive initially
-        let dialog_button = dialog
-            .widget_for_response(ResponseType::Accept)
-            .expect("The dialog needs to have a widget for response type `Accept`.");
-        dialog_button.set_sensitive(false);
+        let cancel_response = "cancel";
+        let create_response = "create";
 
-        // Create entry and add it to the dialog
```

Comment:
Why did you change this comment?

Reply:
I split it to two parts one for creation, other for addition to dialog as addition now is made using `extra_child` in dialog builder and detached from creation code.

And later you suggested to get rid of addition comment [here](https://github.com/gtk-rs/gtk4-rs/pull/1343#discussion_r1148420929).

Reply:
Fair enough :)